controllers/virtualmachinewebconsolerequest/controllers.go

```diff
@@ -7,10 +7,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	"github.com/vmware-tanzu/vm-operator/controllers/virtualmachinewebconsolerequest/v1alpha1"
+	"github.com/vmware-tanzu/vm-operator/controllers/virtualmachinewebconsolerequest/v1alpha2"
 	"github.com/vmware-tanzu/vm-operator/pkg/context"
+	"github.com/vmware-tanzu/vm-operator/pkg/lib"
 )
 
 // AddToManager adds the controller to the provided manager.
 func AddToManager(ctx *context.ControllerManagerContext, mgr manager.Manager) error {
+	if lib.IsVMServiceV1Alpha2FSSEnabled() {
+		return v1alpha2.AddToManager(ctx, mgr)
+	}
 	return v1alpha1.AddToManager(ctx, mgr)
```

Comment:
This controller needs to be added even in the v1a2 enabled world: these are separate types - since the object name has changed - so no version conversion is possible like we do with the other types.

Reply:
good catch. I missed the early return there. I'll return only when the first one(v1a2) errors out.